staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -2104,6 +2109,36 @@ func shouldReleaseExistingOwnedPublicIP(existingPip *network.PublicIPAddress, lb
 		(ipTagRequest.IPTagsRequestedByAnnotation && !areIPTagsEquivalent(currentIPTags, ipTagRequest.IPTags))
 }
 
+//  ensurePIPTagged TODO
+func (az *Cloud) ensurePIPTagged(service *v1.Service, pip *network.PublicIPAddress) bool {
+	changed := false
+	if _, ok := service.Annotations[ServiceAnnotationAzurePIPTags]; !ok {
+		return false
+	}
+	tags := parseTags(service.Annotations[ServiceAnnotationAzurePIPTags])
```

Comment:
need to merge tags from annotations with `config.Tags` (annotation overwrite config)